api/src/pcapi/core/users/email/update.py

```diff
@@ -175,6 +175,20 @@ def confirm_email_update_request(encoded_token: str) -> None:
         )
 
 
+def confirm_email_update_request(encoded_token: str) -> models.User:
+    """Confirm the email update request for the given user"""
+    token = token_utils.Token.load_and_check(encoded_token, token_utils.TokenType.EMAIL_CHANGE_CONFIRMATION)
+    user = models.User.query.get(token.user_id)
+    if not user:
+        raise exceptions.InvalidToken()
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏ça se factorise entre `confirm_email_update_request` et `confirm_email_update_request_and_send_mail`
_low logaf_

Reply:
malheureusement non, les deux fonctions ont besoin de `token` et `user` séparément plus bas :'(